src/frontend/screens/Onboarding/welcome/index.tsx

```diff
@@ -18,61 +19,96 @@ const Welcome: React.FC<WelcomeProps> = function (props) {
-          >
-            {t('hyperplay.onboarding.terms', `Terms of Service.`)}
-          </a>
+            'hyperplay.onboarding.welcome.text.paragraph3',
+            `Seamlessly bring your wallet, tokens, 
+           and assets into every game while accessing 
+           the extensive libraries of the Epic Store, 
+           GOG, and our own HyperPlay Store, all in one place.`
+          )}
         </div>
-        <div>
+        <div className={WelcomeStyles.textBold}>
           {t(
-            'hyperplay.onboarding.welcome.text.bugs',
-            `Please note that HyperPlay is a public alpha. Many features are still
-          in development, and there will be bugs.`
+            'hyperplay.onboarding.welcome.text.bulletTitle',
+            `With HyperPlay, you can:`
           )}
         </div>
+        <div>
+          <ul
+            className={`${WelcomeStyles.bulletPointIcon} ${WelcomeStyles.bulletPointList}`}
+          >
+            <li>
+              <DiamondBlue className={WelcomeStyles.diamondBlue} />
+              {t(
+                'hyperplay.onboarding.welcome.text.bulletPoint_1',
+                `Access a vast collection of 100+ on-chain enabled games`
+              )}
+            </li>
+            <li>
+              <DiamondBlue className={WelcomeStyles.diamondBlue} />
+              {t(
+                'hyperplay.onboarding.welcome.text.bulletPoint_2',
+                `Earn tokens, NFTs, and off-chain rewards by completing Quests`
+              )}
+            </li>
+            <li>
+              <DiamondBlue className={WelcomeStyles.diamondBlue} />
+              {t(
+                'hyperplay.onboarding.welcome.text.bulletPoint_3',
+                `Play Epic and GOG games directly through HyperPlay`
+              )}
+            </li>
+            <li>
+              <DiamondBlue className={WelcomeStyles.diamondBlue} />
+              {t(
+                'hyperplay.onboarding.welcome.text.bulletPoint_4',
+                `Install Steam (Windows version) on macOS with one easy click`
+              )}
+            </li>
+            <li>
+              <DiamondBlue className={WelcomeStyles.diamondBlue} />
+              {t(
+                'hyperplay.onboarding.welcome.text.bulletPoint_5',
+                `Run Windows games on macOS, Linux, and SteamDeck using our
+              advanced`
+              )}{' '}
+              <a
+                onClick={() =>
+                  window.api.openExternalUrl(
+                    `https://docs.hyperplay.xyz/faq#q-what-operating-systems-are-supported`
+                  )
+                }
+              >
+                Compatibility Layer (Beta)
+              </a>
+            </li>
+          </ul>{' '}
+        </div>
         <div>
           {t(
-            'hyperplay.onboarding.welcome.text.communityCTA_1',
-            `We'd love your feedback and to have you join us in our`
+            'hyperplay.onboarding.welcome.text.2',
```

Comment:
for this one, can you not use a number as a key, this might give issues in the future. 
Rename it to something like `intro` for instance.